flows/knack/knack_banner.py

```diff
@@ -4,124 +4,58 @@
 Name: Knack Banner HR App integration
 Description: Update knack HR app based on records in Banner and CTM
 Schedule: "45 13 * * *"
-Labels: atd-data02, knack
 """
```

Comment:
Maybe something for prefect meeting but do we want to include in the docstring how to deployment build and deployment apply for this flow file? 

[I've been doing it](https://github.com/cityofaustin/atd-prefect/blob/c121eaa09f23ea05410e41a6d544ddad335c16be/flows/finance-data/finance_data_to_s3.py#L9) because I kept losing my command and having to retype it all the time.

Reply:
I really like this idea